src/esphome/sensor/mpr121_sensor.cpp

```diff
@@ -0,0 +1,125 @@
+#include "esphome/defines.h"
+#define USE_MPR121 1
```

Comment:
Remove this - it is already set in defines.h (and ESPHome will set it with BUILD_FLAGS)